README.md

```diff
@@ -399,77 +394,72 @@ This topic would preferably be introduced later in the
 "normal" mode, and "add-node" mode.  In normal mode, the
 cursor usually looks like a hand (unless it is near a node), and
 to add a node, you need to hold down the shift key, so the cursor
 turns into crosshairs, before pressing the left mouse button.
 
 In "add-node" mode, things are reversed.  The cursor always
 takes the form of crosshairs, and to add a node, you simply need to
-click with the left mouse button.
+press the left mouse button.
 
 There are two ways to toggle between the two modes.  One way is to
 click on the button, to the right of the `Files` menu, that contains
-a picture of crosshairs.  When this button is light, khartes is in
-normal mode, and when it is dark, khartes is in add-node mode.
+a crosshairs icon.  When this button is enabled, khartes is in
+normal mode, and when it is disabled, khartes is in add-node mode.
 
 The other, more convenient, way to toggle between normal and add-node mode
 is to quickly press and release the shift button.  You can see which
 mode you are in by seeing whether the cursor afterwards is a hand (normal mode)
 or crosshairs (add-node mode).
 
-This topic needed to be introduced now, because you might, by accident, 
+This topic needed to be introduced now, because you might, by accident,
 hit the shift button, unintentionally toggling between modes.
 Now that you understand the two modes, you won't be mystified if this happens.
 
-## Choosing a region for segmentation
+### Choosing a region for segmentation
 
 Before you start creating a fragment, you should make sure that there
 is a region in your high-resolution data volume that is possible to segment.
 
 What you are looking for is a region where the papyrus sheets, as
 seen in the top Data Slice, are not too tilted (you want an angle
-flatter than 45 degrees or so).  This region doesn't have
-to be very large, but if you can't
-find even a small area, then you may
-need to create another high-resolution data volume.
+flatter than 45 degrees or so).  This region doesn't have to be very large,
+but if you can't find even a small area, then you may need to create
+another high-resolution data volume.
 
-## Building and modifying the segment
+### Building and modifying the segment
 
 At this point you are ready to build the segment.
 As you saw in the video, you do this by simply adding nodes to the
 Data Slices.  Notice that **the order you add nodes does not matter**;
 if you feel your nodes are too far apart, simply add more.
 
 But what do you do if you discover that a node is in the wrong
-place?  You have a couple of options.
+place? You have a couple of options.
 
-### Deleting a node
+#### Deleting a node
 
 It is very easy to delete a node.  Simply move the cursor over
 the node, so that the node turns cyan.
 At this point, simply hit the "Delete" or "Backspace" key (either will work).
-But bear this in mind: **There is currently no "undo" in khartes!**
+But bear in mind **deleting a node will remove it permanently**.
```

Comment:
```suggestion
But bear in mind that **deleting a node will remove it permanently**;
it can not be recovered.
```
I have modified it to avoid the 'double negative' "no" and "un-".
Also, let's remove the exclamation mark, as it is exhaustive; the fact it is permanent is already emphasized using **bold** formatting.

Reply:
I agree with the added "that", but I do want to reiterate wherever possible that khartes really does not have an "undo" mode; this is a handy place to do that.

Reply:
```suggestion
But bear in mind **deleting a node will remove it permanently**;
it can not be recovered, as there is currently no undo action implemented.
```

I have added a mention of the undo action, but I have kept the mention of recovery as it helps to keep it in line with the rest of the tutorial.

I should clarify that I also removed the mention of the undo mode because it is already mentioned in https://github.com/KhartesViewer/khartes#things-to-fix , so it may only be temporary. Removing it avoids having to maintain such details. It could alternatively be mentioned once, in bold, at the start of the tutorial.

The added "that" came from your suggested change, but it does look fine.

Reply:
That looks good